pmm-tests/pmm-2-0-bats-tests/generic-tests.bats

```diff
@@ -11,8 +11,8 @@ if [[ $(id -u) -eq 0 ]] ; then
 fi
 run pmm-admin
 echo "$output"
-    [ "$status" -eq 0 ]
-    [ "${lines[0]}" = "usage: pmm-admin [<flags>] <command> [<args> ...]" ]
+    [ "$status" -eq 1 ]
```

Comment:
why does this execution fails?

Reply:
what way are we showing "flags" now?

Reply:
By default the library returns exit code 1 when no command is provided.
Running `pmm-admin --help` prints the help with exit code 0.

Flags are not shown in usage now help now. They can be provided before or after the command.